package.json

```diff
@@ -28,30 +28,42 @@
   ],
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test/*.js --timeout 600000",
-    "test": "c8 mocha",
-    "fix": "eslint '**/*.js' --fix",
+    "lint": "gts fix && eslint --fix samples/*.js",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
+    "test": "c8 mocha build/test",
+    "fix": "gts fix",
     "docs-test": "linkinator docs",
     "predocs-test": "npm run docs",
-    "prelint": "cd samples; npm link ../; npm i"
```

Comment:
I think we want to keep this `prelint` rule to avoid the GitHub action failures in the samples below.  FYI @bcoe @alexander-fenster - do we need to add this to the generator too?

Reply:
So... no :)  Samples do not belong to the generator (yet), so the generator is not supposed to generate scripts it has no idea about. It should belong to samplegen or a Future Magical Publishing Automation that is not yet here.

Reply:
Make sense to me, the [PR](https://github.com/googleapis/nodejs-iot/pull/236/files) I am following removed this line. 

I will add back. @xiaozhenliu-gg5 @alexander-fenster let me know if you have any concern.